### ISSUE TITLE
Permissions error from BigQuery during integration test

### ISSUE THREAD

**Issue by achave11** [DataBiosphere/azul#2161]
~`AZUL_TDR_TARGET` variable is unaccounted for in `environment.py` and in `deployments/.example.local/environment.py`. Which causes IT to fail initially.~ 

When copying `AZUL_TDR_TARGET`  environment variable from the `./example.local` deployment into personal deployment, the following exception is observed:
```
Traceback (most recent call last):
  File "/Users/abrahamchavez/.pyenv/versions/3.8.3/lib/python3.8/unittest/case.py", line 60, in testPartExecutor
    yield
  File "/Users/abrahamchavez/.pyenv/versions/3.8.3/lib/python3.8/unittest/case.py", line 676, in run
    self._callTestMethod(testMethod)
  File "/Users/abrahamchavez/.pyenv/versions/3.8.3/lib/python3.8/unittest/case.py", line 633, in _callTestMethod
    method()
  File "/Users/abrahamchavez/PycharmProjects/Azul/azul.sc/test/azul_test_case.py", line 109, in wrapped
    return method(*args, **kwargs)
  File "/Users/abrahamchavez/PycharmProjects/Azul/azul.sc/test/integration_test.py", line 177, in test
    catalogs: List[Catalog] = [
  File "/Users/abrahamchavez/PycharmProjects/Azul/azul.sc/test/integration_test.py", line 178, in <listcomp>
    Catalog(name=catalog, notifications=self._prepare_notifications(catalog))
  File "/Users/abrahamchavez/PycharmProjects/Azul/azul.sc/test/integration_test.py", line 373, in _prepare_notifications
    bundle_fqids = self.azul_client.list_bundles(catalog)
  File "/Users/abrahamchavez/PycharmProjects/Azul/azul.sc/src/azul/azulclient.py", line 190, in list_bundles
    return self.repository_plugin(catalog).list_bundles(self.prefix)
  File "/Users/abrahamchavez/PycharmProjects/Azul/azul.sc/src/azul/plugins/repository/tdr/__init__.py", line 50, in list_bundles
    bundle_ids = self.client.list_links_ids(prefix)
  File "/Users/abrahamchavez/PycharmProjects/Azul/azul.sc/src/azul/tdr.py", line 283, in list_links_ids
    current_bundles = self._query_latest_version(f'''
  File "/Users/abrahamchavez/PycharmProjects/Azul/azul.sc/src/azul/tdr.py", line 293, in _query_latest_version
    iter_rows = self.big_query_adapter.run_sql(query)
  File "/Users/abrahamchavez/PycharmProjects/Azul/azul.sc/src/azul/bigquery.py", line 58, in run_sql
    return self.client.query(query)
  File "/Users/abrahamchavez/PycharmProjects/Azul/azul.sc/.venv/lib/python3.8/site-packages/google/cloud/bigquery/client.py", line 2361, in query
    query_job._begin(retry=retry, timeout=timeout)
  File "/Users/abrahamchavez/PycharmProjects/Azul/azul.sc/.venv/lib/python3.8/site-packages/google/cloud/bigquery/job.py", line 3151, in _begin
    super(QueryJob, self)._begin(client=client, retry=retry, timeout=timeout)
  File "/Users/abrahamchavez/PycharmProjects/Azul/azul.sc/.venv/lib/python3.8/site-packages/google/cloud/bigquery/job.py", line 639, in _begin
    api_response = client._call_api(
  File "/Users/abrahamchavez/PycharmProjects/Azul/azul.sc/.venv/lib/python3.8/site-packages/google/cloud/bigquery/client.py", line 556, in _call_api
    return call()
  File "/Users/abrahamchavez/PycharmProjects/Azul/azul.sc/.venv/lib/python3.8/site-packages/google/api_core/retry.py", line 281, in retry_wrapped_func
    return retry_target(
  File "/Users/abrahamchavez/PycharmProjects/Azul/azul.sc/.venv/lib/python3.8/site-packages/google/api_core/retry.py", line 184, in retry_target
    return target()
  File "/Users/abrahamchavez/PycharmProjects/Azul/azul.sc/.venv/lib/python3.8/site-packages/google/cloud/_http.py", line 423, in api_request
    raise exceptions.from_http_response(response)
google.api_core.exceptions.Forbidden: 403 POST https://bigquery.googleapis.com/bigquery/v2/projects/broad-jade-dev-data/jobs: Access Denied: Project broad-jade-dev-data: User does not have bigquery.jobs.create permission in project broad-jade-dev-data.

(job ID: 1586072b-fa9d-4f10-912d-6a80c7ff6a95)

         -----Query Job SQL Follows-----          

    |    .    |    .    |    .    |    .    |
   1:
   2:            SELECT links_id, version
   3:            FROM dcp1_metadata_no_files.links
   4:            WHERE STARTS_WITH(links_id, 'e9')
   5:        
    |    .    |    .    |    .    |    .    |

```

**Comment by hannes-ucsc** [comment]
I think the [upgrade instructions](https://github.com/DataBiosphere/azul/blob/develop/UPGRADING.rst#2025-register-indexer-sa-with-broads-sam-during-deployment) for #2025 are incomplete/incorrect. For one environment variables should be copied from the example deployment, not sandbox. More importantly, they should reference the README section (the last two paragraphs of [section 3.2](https://github.com/DataBiosphere/azul#32-provisioning-cloud-infrastructure) as something to be done.

@noah-aviel-dove please fix this in PR #2116.

**Comment by hannes-ucsc** [comment]
Freebie of #2116
